src/Cake.Npm/NpmAliases.cs

```diff
@@ -0,0 +1,359 @@
+namespace Cake.Npm
+{
+    using System;
+    using Core;
+    using Core.Annotations;
+    using Install;
+    using Pack;
+    using RunScript;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Provides a wrapper around Npm functionality within a Cake build script
+    /// </summary>
+    [CakeAliasCategory("Npm")]
+    [CakeNamespaceImport("Cake.Npm.Install")]
+    [CakeNamespaceImport("Cake.Npm.Pack")]
+    [CakeNamespaceImport("Cake.Npm.RunScript")]
+    public static class NpmAliases
```

Comment:
@pascalberger - a question, and the answer may well be related to how cake handles aliases, but....

Would it be better to split the individual Npm commands (Install, Pack, RunScript, etc) into separate alias classes such as `NpmInstallAlias`, `NpmPackAlias` etc?